serverless/src/getRetrievals/handler.js

```diff
@@ -61,12 +61,19 @@ export default async function getRetrievals(event, context) {
         environment
       } = firstRow
 
+      const collectionMetadata = retrievalRecord.map((
```

Comment:
Nit-picky, but you could clean this up very slightly by calling this `collections` instead of `collectionsMetadata`.